lib/validate.test.js

```diff
@@ -166,5 +240,69 @@ describe(__filename, function () {
           done();
         });
     });
+
+    it('should strip the where parameter and accept the list request', function (done) {
+      request
+        .get('/gadgets?who=Sue&where=there')
+        .expect(200)
+        .expect('content-type', /json/)
+        .end(function (err, res) {
+          expect(err).to.not.exist;
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.have.length(0);
+
+          done();
+        });
+    });
+
+    it('should use the default for who and accept the list request', function (done) {
```

Comment:
What is the default for who, I can't see where it is being added. Might be missing something obvious...